lib/std/fs/file.zig

```diff
@@ -314,6 +314,59 @@ pub const File = struct {
         mtime: i128,
         /// Creation time in nanoseconds, relative to UTC 1970-01-01.
         ctime: i128,
+
+        const Self = @This();
```

Comment:
The existing references to Stat in this object use it's name rather than Self.  I think `@This()` is only necessary in generics or in files that are structs, like [std.mem.Allocator](https://github.com/ziglang/zig/blob/master/lib/std/mem/Allocator.zig#L7).

That said, I think this still qualifies as idiomatic zig, I'm just griping about the mixed use of `Stat` and `Self`.

```suggestion
```